server/upstream/upstream-packages-match.txt

```diff
@@ -95,6 +95,7 @@ bombermaze:
 bot-sentry:
 brasero:
 brltty:
+buildstream:BuildStream
```

Comment:
The file is 'upstreamname:opensusepackagename' - so this is the wrong way around